src/utils/noteServerCalls.js

```diff
@@ -0,0 +1,114 @@
+import axios from "axios";
+import { defaultNewNoteData } from "../components/Note/defaultNewNoteData";
+// import { isPresentInList } from "./helperFunctions";
```

Comment:
comments should be removed

Reply:
Will surely do that in next PRs. Thanks!